doc/generate_config_rst.py

```diff
@@ -0,0 +1,146 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the Apache 2.0 License.
+
+import os
+import sys
+import json
+import tempfile
+import filecmp
+
+# Generated document is included in existing page, so
+# start at heading of depth 1 (equivalent to markdown h2.)
+START_DEPTH = 1
+
+
+class MinimalRstGenerator:
+    def __init__(self):
+        self._lines = [".."]
+        self._lines.append("  This is an auto-generated file. DO NOT EDIT.\n")
+
+    def _add_lines(self, lines):
+        self._lines.extend(lines)
+        self._lines.append("\n")
+
+    def add_heading(self, text, depth):
+        depth_to_char = {0: "=", 1: "-", 2: "~", 3: "+"}
+        self._add_lines([text, depth_to_char[depth] * len(text)])
+
+    def add_line(self, text):
+        self._add_lines([text])
+
+    def render(self):
+        return "\n".join(self._lines)
+
+
+def print_attributes(entry):
+    def stringify_output(s):
+        return f"``{json.dumps(s)}``"
+
+    desc = ""
+    if "description" in entry:
+        desc += entry["description"]
+    if "enum" in entry:
+        desc += (
+            f' (values: {", ".join(map(lambda s: stringify_output(s), entry["enum"]))})'
```

Comment:
```suggestion
            f' (values: {", ".join(stringify_output(s) for s in entry["enum"])})'
```

is clearer in my opinion.